sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -319,9 +347,10 @@ object Klaviyo {
             }
         }
 
-        getPushToken()?.let { event[EventKey.PUSH_TOKEN] = it }
+        UserInfo.pushToken.let { event[EventKey.PUSH_TOKEN] = it }
 
-        createEvent(event)
+        Registry.log.verbose("Enqueuing ${event.metric.name} event")
+        Registry.get<ApiClient>().enqueueEvent(event, UserInfo.getAsProfile())
```

Comment:
why not keep `createEvent` here?

Reply:
If i used`createEvent` or `getPushToken` in this method, those are wrapped in `safeCall` and when they hit an exception, it gets swallowed instead of bubbling up to handlePush's `safeApply`

Reply:
ahh ok i see that now